config/mongoid.yml

```diff
@@ -9,6 +9,7 @@ common: &default_client
     timeout: <%= ENV['MONGOID_TIMEOUT'] || 0.5 %>
     ssl: <%= ENV['MONGOID_USE_SSL'] || false %>
     auth_source: <%= ENV['MONGOID_AUTH_SOURCE'] || '' %>
+    auth_mech: <%= ENV['MONGOID_AUTH_MECH'].nil? ? ':scram' : '' %>
```

Comment:
@pomegranited I think this wont work for you.  Maybe this?

```suggestion
    auth_mech: <%= ENV['MONGOID_AUTH_MECH'].nil? ? ':scram' : ENV['MONGOID_AUTH_MECH'] %>
```

Reply:
Sigh.. you're totally right @bderusha .  Thank you!